compiler/src/iree/compiler/Dialect/Flow/Transforms/test/CMakeLists.txt

```diff
@@ -24,7 +24,6 @@ iree_lit_test_suite(
     "dispatch_linalg_on_tensors.mlir"
     "dispatch_linalg_on_tensors_fusion.mlir"
     "dispatch_linalg_on_tensors_fusion_with_transpose.mlir"
-    "dispatch_linalg_transform_dialect.mlir"
```

Comment:
couldn't those be marked XFAIL rather than a full deletion from the build files ?